utils/src/parallel_restore.c

```diff
@@ -1146,6 +1154,13 @@ static spr_err_t prepare_alloc_builders(struct scoutfs_parallel_restore_writer *
 	if (err)
 		goto out;
 	wri->meta_len -= len;
+	/* store reserved extra space a free extent*/
+	end = start + len;
+	extra = le64_to_cpu(wri->super.total_meta_blocks) / EXTRA_BLOCK_SLICE;
+	err = insert_free_items(&wri->meta_btb[0], end, extra);
```

Comment:
could just write `start + len` inside the function call here and avoid needing to declare `end` entirely and pre-calculate.